tools/cldr-code/src/test/java/org/unicode/cldr/util/TestCLDRFile.java

```diff
@@ -64,4 +66,26 @@ public void TestDraftStatus(DraftStatus status) {
             () -> assertEquals(status, DraftStatus.forString(status.name().toUpperCase()))
         );
     }
+
+    @Test
+    public void TestReadBCP47() {
+        Path bcp47Path = CLDRConfig.getInstance().getCldrBaseDirectory().toPath().resolve("common/bcp47");
+        Factory factory = Factory.make(bcp47Path.toString(), ".*");
+        assertNotNull(factory);
+        for (final String id : factory.getAvailable()) {
+            System.out.println(id);
+            CLDRFile file = factory.make(id, false);
+            assertNotNull(file, id);
+            for (final String xpath : file.fullIterable()) {
+                assertNotNull(xpath, id + ":" + xpath);
```

Comment:
This will assert that id + ":" + xpath is not null. That will always be the case. Same below. I think the arguments are reversed.

Reply:
This test uses the standard `org.junit.jupiter.api.Assertions.assertNotNull` which has the args in the other order.

Reply:
anyway, the iterator itself was failing previously. Working with @nordzilla to repro.